polkadot-parachain/src/command.rs

```diff
@@ -695,24 +696,19 @@ pub fn run() -> Result<()> {
 
 			// Switch on the concrete benchmark sub-command-
 			match cmd {
+				#[cfg(not(feature = "runtime-benchmarks"))]
+				BenchmarkCmd::Pallet(_) => Err("Benchmarking wasn't enabled when building the node. \
+					You can enable it with `--features runtime-benchmarks`."
+					.into()),
+				#[cfg(feature = "runtime-benchmarks")]
 				BenchmarkCmd::Pallet(cmd) =>
-					if cfg!(feature = "runtime-benchmarks") {
```

Comment:
I think its a bit easier to read with the `cfg!` macro. Why the change?

Reply:
I don't see any big difference. The only reason why I changed this is that it is done with attributes in all other places. So, just didn't want to mix multiple approaches.

Reply:
I dont have a strong opinion about it - just wanted to comment.